client/hbs/not-found.hbs

```diff
@@ -1 +1,15 @@
-Ya dun goofed. Go <a href="/">home</a>?
+<html>
+{{> head.hbs}}
+
+<body>
+	{{> header.hbs}}
+	<content-row>
+		<main>
+			<p>Hmmm... looks like we couldn't find that.</p>
+			<p>If we sent you here, please <a href="mailto:webmaster@dttw.tech">let us know</a>.</p>
```

Comment:
I'm probably just going to file an issue for making a real 404, because there's no reason to worry too much about it right now.